README.md

```diff
@@ -7,361 +8,1062 @@
+
+Accepts the file ID and fetches the details as per
+the [API documentation here](https://docs.imagekit.io/api-reference/media-api/get-file-details)
 
 ```python
-imagekit.get_file_details(file_id)
+file_id = "your_file_id"
+result = imagekit.get_file_details(file_identifier=file_id)  # fileId required
+
+# Final Result
+print(result)
+
+# Raw Response
+print(result.response_metadata.raw)
+
+# print that file's id
+print(result.file_id)
 ```
 
-**3. Get File Metadata**
-Accepts the file ID and fetches the metadata as per the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-for-uploaded-media-files)
+**3. Get File Versions**
+
+Accepts the file ID and fetches the details as per
+the [API documentation here](https://docs.imagekit.io/api-reference/media-api/get-file-versions)
+
 ```python
-imagekit.get_file_metadata(file_id)
+file_id = "your_file_id"
+result = imagekit.get_file_versions(file_identifier=file_id)  # fileId required
+
+# Final Result
+print(result)
+
+# Raw Response
+print(result.response_metadata.raw)
+
+# print that file's version id
+print(result.list[0].version_info.id)
 ```
 
+**4. Get File Version details**
 
-**3. Get File Metadata from remote url**
-Accepts the remote file url and fetches the metadata as per the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-from-remote-url)
+Accepts the `file_id` and `version_id` and fetches the details as per
+the [API documentation here](https://docs.imagekit.io/api-reference/media-api/get-file-version-details)
 
 ```python
-imagekit.get_remote_file_url_metadata(remote_file_url)
+result = imagekit.get_file_version_details(file_identifier='file_id',
+        version_identifier='version_id')
+
+# Final Result
+print result
+
+# Raw Response
+print result.response_metadata.raw
+
+# print that file's id
+print result.file_id
+
+# print that file's version id
+print result.version_info.id
 ```
 
-**4. Update File Details**
-Update parameters associated with the file as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/update-file-details).
-The first argument to the `update_field_details` method is the file ID, and a second argument is an object with the
-parameters to be updated.
+**5. Update File Details**
+
+Accepts all the parameters as per
+the [API documentation here](https://docs.imagekit.io/api-reference/media-api/update-file-details).
+The first argument to the `update_file_details()` method is the file ID, and a second argument is an object with the
+parameters to be
+updated.
 
 ```python
-imagekit.update_file_details("file_id", {
-    "tags": ["image_tag"],
-    "custom_coordinates": "10,10,100, 100"
-})
+from imagekitio.models.UpdateFileRequestOptions import UpdateFileRequestOptions
+
+options = UpdateFileRequestOptions(
+    remove_a_i_tags=['remove-ai-tag-1', 'remove-ai-tag-2'],
```

Comment:
@imagekitio internally we have snake_to_lower_case automatically transform all request parameters:
https://github.com/imagekit-developer/imagekit-python/blob/b89c2438e923d91b8754bc04726d6cad1e6754cb/imagekitio/file.py#L117
https://github.com/imagekit-developer/imagekit-python/blob/b89c2438e923d91b8754bc04726d6cad1e6754cb/imagekitio/utils/formatter.py#L30

So, now if we use remove_ai_tags it'll send removeAiTags to server instead of the intended removeAITags. We can add special case handling but it'll make it a little inconsistent.

Reply:
We use `remove_ai_tags` in other SDKs as well. `ai` comes together.

Reply:
@imagekitio - We have updated it to us `remove_ai_tags` across code / readme / quickstart / docs PRs. 

--

Test case for same is here, 
https://github.com/amanforindia/imagekit-python/blob/483404f1e45706c5e2281c61b038aa1cca4d4859/tests/test_files_ops.py#L1423